### ISSUE TITLE
Python driver unable to build from source

### ISSUE THREAD

**Issue by Hunaid2000** [apache/age#388]
**Describe the bug**
When building from source (following instructions in Readme) it says 
`Exception: Could not deserialize ATN with version ♥ (expected 4)`

Unable to build when package antlr4-python3-runtime installed is version 4.10 or above

The issue was also referred in apache-age-python PyPi package repository rhizome-ai/apache-age-python#18

**How are you accessing AGE (Command line, driver, etc.)?**
- Command line
- Python Driver

**What data setup do we need to do?**
- Install all required packages given in setup.py or Readme (psycopg2 and antlr4-python3-runtime)

**What is the necessary configuration info needed?**
- Install antlr4-python3-runtime version 4.10 or above to test the error

**What is the command that caused the error?**
```python
python setup.py install
```

```
Traceback (most recent call last):
  File "C:\Users\hunai\Desktop\AGE\age\drivers\python\setup.py", line 17, in <module>
    from age import VERSION
  File "C:\Users\hunai\Desktop\AGE\age\drivers\python\age\__init__.py", line 16, in <module>
    from . import age
  File "C:\Users\hunai\Desktop\AGE\age\drivers\python\age\age.py", line 22, in <module>    from .builder import ResultHandler , parseAgeValue, newResultHandler
  File "C:\Users\hunai\Desktop\AGE\age\drivers\python\age\builder.py", line 16, in <module>
    from .gen.AgtypeLexer import AgtypeLexer
  File "C:\Users\hunai\Desktop\AGE\age\drivers\python\age\gen\AgtypeLexer.py", line 106, in <module>
    class AgtypeLexer(Lexer):
  File "C:\Users\hunai\Desktop\AGE\age\drivers\python\age\gen\AgtypeLexer.py", line 108, in AgtypeLexer
    atn = ATNDeserializer().deserialize(serializedATN())
  File "C:\Users\hunai\AppData\Local\Programs\Python\Python310\lib\site-packages\antlr4\atn\ATNDeserializer.py", line 28, in deserialize
    self.checkVersion()
  File "C:\Users\hunai\AppData\Local\Programs\Python\Python310\lib\site-packages\antlr4\atn\ATNDeserializer.py", line 50, in checkVersion
    raise Exception("Could not deserialize ATN with version " + str(version) + " (expected " + str(SERIALIZED_VERSION) + ").")
Exception: Could not deserialize ATN with version ♥ (expected 4).
```

**Expected behavior**
Should be able to successfully build from source.

**Environment (please complete the following information):**
- Version: 1.1.0

Documentation of [apache-age-python](https://pypi.org/project/apache-age-python/ ) package should also be updated.

**Comment by jrgemignani** [comment]
@dehowef Can you take a look as you have been working on the Python driver?

**Comment by dehowef** [comment]
Hello, yeah this is an issue with the Python driver. You need a specific version of ANTLR for it to work-- I think the version is 4.9.2. It's in the documentation, you have to use:

pip install antlr4-python3-runtime==4.9.2


Hope this helps

@jrgemignani @Hunaid2000

**Comment by Hunaid2000** [comment]
Thanks for your reply @dehowef. 

How about if we move all dependencies/packages to requirements.txt file? It would make it easier for user to install all packages with one line and give developers more control over required packages (along with their required versions). Any guidance on this would be helpful.

I submitted a PR (mentioned in my earlier comment) that contains following changes:
- Resolves this issue (with requirements.txt)
- Fixed typo in exceptions.py
- Fix ambiguity in README (giving user instruction to run tests and then cloning the repo. Should have been opposite)

@dehowef Kindly have look into it and let me know if I should close this issue and PR now.

**Comment by Hunaid2000** [comment]
Resolved in #389.
